components/mediation-ui/org.wso2.carbon.rest.api.ui/src/main/resources/web/api/sourceview_api.jsp

```diff
@@ -121,6 +121,18 @@
         var isValidXML = isValidXml(trim(source));
         if (!isValidXML) {
             return false;
+        } else {
```

Comment:
I tested this with Firefox, Chrome and Safari and working fine these three browsers.